### ISSUE TITLE
Check new branch

### ISSUE THREAD

**Issue by RyanGoslingsBugle** [FedericaLucia/Gotcha#1]
Okay, I've made some comments on your code, should solve the issues you're having.

https://github.com/FedericaLucia/Gotcha/commit/65ecd58006ca6764fb5c45ff27c7ef06ac3d7522

Ignore the project settings stuff.

**Comment by FedericaLucia** [comment]
Wow, this whole gh thing is new to me. Will see how to merge the project first. Cannot thank you enough! :)

**Comment by RyanGoslingsBugle** [comment]
Probably not worth merging in the new branch, just make the changes to master - merging can get pretty tricky to keep straight.

**Comment by FedericaLucia** [comment]
Firstly thanks a lot! I can see the first picture of the petitions in the About page, but the rest is still broken sadly. Moreso, the js code that makes certain features work (modal button to read more of the petition, the login etc) don't work any longer. I can totally take the blame as I must have merged the branch you created. As I said it's my very first time working on Visual Studio and github, so sorry for the unnecessary pain.

**Comment by RyanGoslingsBugle** [comment]
Yep, read the comments in the _layout and about pages, you'll see why.

**Comment by FedericaLucia** [comment]
Did you get the same errors? Otherwise, it's entirely my fault by merging the branches the wrong way.

**Comment by FedericaLucia** [comment]
Oh cool.

**Comment by FedericaLucia** [comment]
..think I got it..? Did I just have to remove the comments? :D Oh I feel like a lost case.

**Comment by RyanGoslingsBugle** [comment]
Nah, so, if you look at the comment I made in the about page, it explains why your images aren't loading. Just the links in all of them like the first one that I did. In layout, look at how jquery is being loaded - you can either put that in a partial view or let it load as it was (I'm suggesting you use a partial).

**Comment by FedericaLucia** [comment]
Got it! Now it works, I just had to read your comments, which are superb! Rich, thank you so so much!

**Comment by FedericaLucia** [comment]
p.s I am still laughing when I wrote > I just remove your comments. haaa! All the best!

**Comment by FedericaLucia** [comment]
This one is a little better: https://github.com/FedericaLucia/Gotcha-Master.git. Still messy code, not very DRY and loads of errors here and there, nonetheless I wouldn't have made it till this point if not for your help! Thank you again, Rich! :)
